src/app/tabs/runTab/model/dropdownlogic.js

```diff
@@ -7,6 +7,7 @@ var typeConversion = remixLib.execution.typeConversion
 var txExecution = remixLib.execution.txExecution
 var CompilerAbstract = require('../../../compiler/compiler-abstract')
 var EventManager = remixLib.EventManager
+var Web3 = require('web3')
```

Comment:
It's usually better for tree shaking to be more precise about the import: `const { utils } from 'web3'` or with require `const { utils } = require('web3')`